README.md

```diff
@@ -4,7 +4,14 @@
 
 > React Bookstore is a simulation book Library where a user can add a book where a title is given and a category is given. The data is then displayed as a list of books on the homepage. 
```

Comment:
I'm selecting this file because there are no CSS changes in this pull request.

## 1.

- [ ] Please, kindly align this section as featured in the Zeplin design:

Your Implementation | Zeplin Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/154943836-ed6d9c65-495b-4bb2-8c1b-4149bddaeaa5.png) | ![image](https://user-images.githubusercontent.com/57421823/154943896-cf76859e-14a8-4258-9966-8ba086ec6155.png)

Kindly please align this section, so it matches the alignment of the last section, like in the Zeplin 💪 


## 2.


- [ ] In the files `navbar.module.css`, `bookItem.module.css`, and `addBook.module.css`, there are multiple instances where you repeat the same rule several times for multiple declarations on the same file. 

 One example is this one in `bookItem.module.css`:

![image](https://user-images.githubusercontent.com/57421823/154946347-af7ae7d4-a756-42c5-8d56-de57ffd5915e.png)

As explained [here](https://speckyboy.com/good-bad-css-practices/), you can correct these files in 2 ways:

1. 
![image](https://user-images.githubusercontent.com/57421823/154946799-74e2b04f-e285-45d1-b334-648ee7b13c60.png)

2. 
![image](https://user-images.githubusercontent.com/57421823/154946893-0ba2699d-64cd-4484-828f-fdf1d30b42af.png)

The files that need corrections to avoid rule repetitions are `navbar.module.css`, `bookItem.module.css`, and `addBook.module.css`.  There are not many rules, (mainly font rules), but those are repeated several times on the same file.

If you implement this change, your CSS modules will be cleaner, follow best practices, and will be a lot more readable.  Like a pro. 💪